pkg/aws/resources_container.go

```diff
@@ -62,7 +62,7 @@ func (resources *ResourcesContainer) AllRegions() []string {
 }
 
 func (resources *ResourcesContainer) GetResources() common.ResourcesModel {
-	return resources
+	return &resources
```

Comment:
```suggestion
	return resources
```
`resources` is already a pointer